selfdrive/car/docs_definitions.py

```diff
@@ -31,60 +30,83 @@ class Star(Enum):
   EMPTY = "empty"
 
 
-class Harness(Enum):
```

Comment:
How about you have dataclasses for each part type, and one big enum to store everything in?

something like:

```python
@dataclass
class BasePart:
  name: str

  def __repr__(self):
    return self.name

  def __str__(self):
    return self.name

  @property
  def type(self) -> str:
    raise NotImplementedError


@dataclass
class Connector(BasePart):
  @property
  def type(self) -> str:
    return 'connector'


@dataclass
class Mount(BasePart):
  @property
  def type(self) -> str:
    return 'mount'


class CarPart(Enum):
  nidec = Connector("Honda Nidec connector")
  ...
  mount = Mount("mount")
  ...
```